lib/LANraragi/Utils/Archive.pm

```diff
@@ -360,7 +360,7 @@ sub extract_file_from_archive ( $archive, $filename ) {
     my $tmp = File::Temp->new( DIR => $path );
     $tmp->unlink_on_destroy(0);
```

Comment:
```suggestion
    my $tmp = File::Temp->tempdir( DIR => $path, CLEANUP => 1 );
```

Reply:
Right i didn't really understand what it does, this sounds like it works.
Not sure if its my install but i get 

> Error: 'tempdir' can't be called as a method

Reply:
ah, this also needs the `use` statement at the top of the module to add `tempdir` alongside `tempfile`.

Reply:
huh nvm it's already there, I messed up and this doesn't need the OOP interface so:
`my $tmp = tempdir( DIR => $path, CLEANUP => 1 );` 
should work instead.

Reply:
Thanks! Seems to work as expected again now.